src/common/containers/build-details.js

```diff
@@ -72,9 +71,12 @@ class BuildDetails extends Component {
         <BetaNotification />
         <Breadcrumbs>
           { user &&
-            <Link to={`/user/${user.login}`}>My repos</Link>
+            <BreadcrumbsLink to={`/user/${user.login}`}>My repos</BreadcrumbsLink>
           }
-          <Link to={`/user/${repository.fullName}`}>{repository.fullName}</Link>
+          {' ' /* vanilla assumes space between breadcrumb elements */}
+          <BreadcrumbsLink to={`/user/${repository.fullName}`}>{repository.fullName}</BreadcrumbsLink>
+          {' ' /* vanilla assumes space between breadcrumb elements */}
+          <BreadcrumbsLink>&nbsp;</BreadcrumbsLink> {/* hack to show arrow after last breadcrumb item */}
```

Comment:
I can see that the intent is so the breadcrumb links to the title of the page. I feel we should still include this in the breadcrumb but not as something that can be interacted with? 

This is a common pattern across our sites and I feel we should do the same in Snapcraft. @carlaberkers what are you thoughts? I've attached a screenshot of what it looks like with the new VF updates.

![image](https://user-images.githubusercontent.com/1201618/27859859-15433f38-6172-11e7-8861-658444036d84.png)